Docs: Minor change missed in #14590 (#14604)
apache/druid: docs/configuration/index.md

##### Smart segment loading

The `smartSegmentLoading` mode simplifies Coordinator configuration for segment loading and balancing.
If you enable this mode, do not provide values for the properties in the table below as the Coordinator computes them automatically.

> The values are computed based on the current state of the cluster and are designed to optimize Coordinator performance.

Reviewer: ```suggestion
Druid computes the values to optimize Coordinator performance, based on the current state of the cluster.
```
